howso/analysis.amlg

```diff
@@ -538,10 +538,15 @@
 			)
 			;set the next auto analysis threshold
 			(assign_to_entities (assoc
-				!autoAnalyzeThreshold (* num_cases !autoAnalyzeGrowthFactorAmount)
+				!autoAnalyzeThreshold
```

Comment:
what's the logic for this change?  this breaks normal logic for large batch training. ie, you threshold is at 100 but the first batch trains 1000, which is already bigger than the threshold, it'll analyze but then immediatlly set it to be analyzed again, which may not be necessary if the next batch only trains 10.

Reply:
The logic was an attempt to turn this into a delta. Clearly there's more work to be done on this front before that change can be switched over in its entirety, as your other comment alludes to.